docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -433,6 +433,10 @@ dependencyManagement {
 ----
 ====
 
+=== DefaultOpenForTrafficCount and its effect on EurekaServer warmup time
+
+In order to enable the warmup time, Set `eureka.server.defaultOpenForTrafficCount=0`,It enables the warmup time, and the load testing app never gets its local cache emptied once the server comes back online.But expectedNumberOfRenewsThreshold will always be at 0 and never get updated for a standalone server.Also, Netflix Eureka's waitTimeInMsWhenSyncEmpty setting is not taken into account in Spring Cloud Eureka server at the beginning.
```

Comment:
Please move the information that Netflix Eureka's waitTimeInMsWhenSyncEmpty setting is not taken into account in Spring Cloud Eureka server to the beginning of the paragraph.

Reply:
Please remove this part:  "and the load testing app never gets its local cache emptied once the server comes back online" - it's just information on how the user has tested it.

Reply:
Please switch from "It enables the warmup time," to "It will enable the warmup time." (also end the sentence with a full stop and also put a full stop before it).